api/envoy/config/filter/http/ext_authz/v2/ext_authz.proto

```diff
@@ -52,6 +52,8 @@ message ExtAuthz {
   bool use_alpha = 4 [deprecated = true];
 
   // Enables filter to buffer the client request body and send it within the authorization request.
+  // A ``x-envoy-auth-partial-body: 0|1`` metadata header will be added to the authorization
```

Comment:
Personally I would do "true" and "false" here vs. "0" and "1" to clearly indicate boolean, but I don't feel strongly about it. WDYT? A more general question would be why use a header at all vs. adding a boolean field to the request proto?

Reply:
I suggested using headers here for a couple of reasons. Please see https://github.com/envoyproxy/envoy/pull/6583#issuecomment-483764384 for details.

Reply:
@mattklein123 `true|false` vs `0|1` - I have no preference. The bool values are much clearer, so I can change that.

Reply:
If you have all already discussed headers vs. field and determined header is better that's fine with me, but yeah let's change to true/false if you don't mind. Thank you!

/wait

Reply:
@mattklein123 changed. Thanks.